test/aws/scaleup.yml

```diff
@@ -13,13 +13,11 @@
     command: mktemp -d /tmp/openshift-ansible-XXXXXXX
     register: mktemp
     changed_when: False
-  - name: add localhost as master
-    add_host:
-      name: localhost
-      ansible_connection: local
-      groups: masters
 
   - import_tasks: ssh_bastion.yml
+    vars:
+      ssh_bastion_namespace: "openshift-ssh-bastion"
```

Comment:
These are test playbooks, not sure if this needs to be configurable

Reply:
I'll remove the var.